.gitlab/e2e.yml

```diff
@@ -169,7 +172,7 @@ new-e2e-containers-dev:
 
 new-e2e-containers-main:
   extends: .new_e2e_template
-  rules: !reference [.on_main]
+  rules: !reference [.on_main_and_no_skip_e2e]
```

Comment:
Note for reviewers:

```yaml
.on_main:
  - <<: *if_main_branch
```

Reply:
When the `when` keyword is not specified, it defaults to `on_success`. What's the reason to change this to `when: always`?

Reply:
Changing it !